src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -73,4 +79,12 @@ class ComService(
 
     return com
   }
+
+  fun searchForOffenderOnStaffCaseload(body: ProbationUserSearchRequest): List<OffenderResult> {
+    return probationSearchApiClient.searchLicenceCaseloadByTeam(body.query, getTeamCodesForUser(body.staffIdentifier))
```

Comment:
there should be a mapping level from the `OffenderResult` to an API model with appropriate openapi annotations. If you look at how LicenceService uses TransformerFunctions hopefully that makes sense.